test/performance/odp_l2fwd.c

```diff
@@ -2035,13 +2052,30 @@ int main(int argc, char *argv[])
 		return -1;
 	}
 
-	pkt_len = POOL_PKT_LEN;
+	pkt_len = gbl_args->appl.packet_len;
 
 	if (pool_capa.pkt.max_len && pkt_len > pool_capa.pkt.max_len) {
 		pkt_len = pool_capa.pkt.max_len;
 		printf("\nWarning: packet length reduced to %u\n\n", pkt_len);
 	}
 
+	seg_len = gbl_args->appl.seg_len;
+
+	/* Check weather we have sufficient segments to support requested packet
+	 * length, if not adjust to bigger segment size */
+	if ((seg_len * pool_capa.pkt.max_segs_per_pkt) < pkt_len)
+		seg_len = pkt_len / pool_capa.pkt.max_segs_per_pkt;
```

Comment:
It would be clearer to use the same calculation for the condition, i.e. if (seg_len < pkt_len / pool_capa.pkt.max_segs_per_pkt).

Reply:
Will change if condition. Regarding warning at line number 2076 we are printing both requested and configured values for user reference